tinygrad/runtime/ops_clang.py

```diff
@@ -1,28 +1,47 @@
-import ctypes, subprocess, pathlib, tempfile
+import os, platform, subprocess, ctypes, pickle
+from mmap import PROT_READ, PROT_WRITE, PROT_EXEC, MAP_ANON, MAP_PRIVATE
 from tinygrad.device import Compiled, Compiler, MallocAllocator
-from tinygrad.helpers import cpu_time_execution, DEBUG, cpu_objdump
+from tinygrad.runtime.support.elf_linker import fixup_relocations
+from tinygrad.helpers import cpu_time_execution, OSX
 from tinygrad.renderer.cstyle import ClangRenderer
 
+libc = ctypes.CDLL(None)
```

Comment:
See how this is done elsewhere in repo

Reply:
replaced None with ctypes.util.find_library('c'). Can't use runtime.autogen.libc because it only exports linux stuff and i need mac stuff like pthread_jit_write_protect_np and sys_icache_invalidate too